### ISSUE TITLE
Gruntfile cannot be parsed under Windows

### ISSUE THREAD

**Issue by softwords** [palantir/plottable#2815]
Visual Studio 2015 provides a range of new tools for web development, including support for node, npm, bower, grunt.
Experiments suggest the best way to work with plottable in VS2015 is to open its folder as a "Web Site" - this gives you access to edit the files in the folder without creating an unnecessary project file. You use the tasks in the gruntfile (accessed from the new Task Runner Explorer) in place of project-level build processes.
However, currently, gruntfile does not parse under Windows. The Task Runner Explorer in VS2015 shows "No tasks found".
The issue is here:
```javascript
 var updateTsFiles = function() {
    tsFiles = grunt.file.read("src/reference.ts")
      .split("\n")
```

This fails under Windows because line ends are now ```\r\n```

A solution ( taken from [here](http://stackoverflow.com/questions/6831918/node-js-read-a-text-file-into-an-array-each-line-an-item-in-the-array)) is simply to add this:

```javascript
  var updateTsFiles = function() {
    tsFiles = grunt.file.read("src/reference.ts")
      .replace(/\r\n/g,"\n")
      .split("\n")
```

**Comment by jtlan** [comment]
@softwords Was this fixed by #2816?

**Comment by softwords** [comment]
Yes, but, this code is now gone due to the restructure of the compile process to exploit the options of the compiler. So the only issue  remaining under windows (also relating to line-ends) is #2914.
